test/spec/pages/login_page.rb

```diff
@@ -78,6 +78,12 @@ def toggle_form_mode
   end
 
   def confirm_email(email) # TODO: test real email confirmation flow
-    request_api('/test/confirm_user', { email: email })
+    if @config.key?('proxy')
+      addr = @config['self_url'].sub 'test.', 'check-api.test.'
```

Comment:
No, we can't. It is just for this case in particular.
The alternative is to add a new key in the config file.